src/main/java/com/todoist/pojo/HabitPushNotifications.kt

```diff
@@ -0,0 +1,3 @@
+package com.todoist.pojo
+
+open class HabitPushNotifications<F: NotificationFeature>(open val features: List<F>)
```

Comment:
This seems to be map instead of a list in the JSON, doesn't it?

```
   "features": {
        "plan_your_day": {"enabled": bool, "send_at": int | null},
        "review_your_day": {"enabled": bool, "send_at": int | null},
        "onboarding_tips": {"enabled": bool, "send_at": int | null}
    }
```

Reply:
It was changed to an array. The spec is outdated but I'd update it.